PlayDate/Views/Cells/MatchTableViewCell.swift

```diff
@@ -8,16 +8,89 @@
+        
+        return imageView
+    }()
+    
+    private let petNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 21, weight: .semibold)
+        //label.text = appUser.pets
+        return label
+    }()
+    
+    private let ownerNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 19, weight: .regular)
+        //label.text = appUser.name
+        return label
+    }()
+    
+    private let descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 15, weight: .regular)
+        label.textColor = .lightGray
+        label.numberOfLines = 0
+        //label.text = appUser.bio
+        return label
+    }()
+    
+//    private let noMatchesLabel: UILabel = {
+//        let label = UILabel()
+//        label.text = "No Matches"
+//        label.textAlignment = .center
+//        label.textColor = .gray
+//        label.font = .systemFont(ofSize: 21, weight: .medium)
+//
+//        return label
+//    }()
 
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.addSubview(profileImageView)
+        contentView.addSubview(petNameLabel)
+        contentView.addSubview(ownerNameLabel)
+        contentView.addSubview(descriptionLabel)
     }
 
-    override func setSelected(_ selected: Bool, animated: Bool) {
-        super.setSelected(selected, animated: animated)
-
-        // Configure the view for the selected state
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        profileImageView.frame = CGRect(x: 5, y: 0, width: 100, height: 100)
+        
+        petNameLabel.frame = CGRect(x: profileImageView.frame.size.width + 20,
+                                      y: 10,
+                                      width: contentView.frame.width - 20 - profileImageView.frame.width,
+                                      height: (contentView.frame.height - 20) / 2)
 
+        ownerNameLabel.frame = CGRect(x: profileImageView.frame.size.width + 100,
+                                      y: 10,
+                                      width: contentView.frame.width - 80 - profileImageView.frame.width,
+                                      height: (contentView.frame.height - 20) / 2)
+        
+        descriptionLabel.frame = CGRect(x: profileImageView.frame.size.width + 20,
+                                        y: ownerNameLabel.frame.size.height + 10,
+                                        width: contentView.frame.width - 20 - profileImageView.frame.size.width,
+                                        height: (contentView.frame.height - 20) / 2)
+    }
```

Comment:
We might have to convert these to constraints later, but we can do some testing to figure it out once we actually have data we can use. Awesome job!

Reply:
> Nice job! Can you address the comments, and after that, feel free to merge!

Right, sorry! I meant to add a comment about it after I made the pull request but it slipped my mind. I added those lines after taking the screenshot. I wanted to have it set up so that I wasn't relying on hard coded strings for the labels, but I couldn't manage to implement it correctly without an error popping up. I commented it out for now so that you guys can still run the app without any errors. I can definitely take it out if it's unnecessary though.